Title: About
Slug: about
Date: 2025-08-02

Prof. Dr. Eduardo Gade Gusmao is a Brazilian bioinformatician, professor, and writer.
This site hosts his lab notes, blog posts, creative experiments, and tools for computer science, biological science, and beyond.

As a computational biologist with a strong focus on the integration of machine/deep learning, probabilistic modeling, and high-dimensional multi-omics biological data to elucidate the molecular and chromatin-level mechanisms of aging. His **primary research interest lies in the study of chromatin conformation and its role in gene regulation, with the goal of developing Chromatin Architecture Biomarkers (CABs) that can serve as predictive markers for aging and age-related diseases.** This work bridges computational and experimental approaches to investigate the intricate relationships between chromatin structure, epigenetic regulation, and gene expression in both normal and pathological cellular states.

To this end, he aims to advance methods that leverage hypergraph-based and deep probabilistic learning techniques to capture and model the spatial organization of chromatin. **His vision is to establish a novel class of CABs that integrate insights from multi-omics datasets, enabling precise biomarker identification for aging interventions and immuno-celular therapies.** With a foundation in both computational biology and molecular biology, and a growing expertise in laboratory techniques, he is well-positioned to develop innovative frameworks that unify large-scale multi-omics data, chromatin architecture studies, and precision medicine approaches to advance the understanding of cellular aging and to foster translational applications in therapeutic and pharmacogenomics research.

You can view his brief CV below (all dates in decreasing order) and his full CV at: [Download my CV (PDF)]({static}/CV.pdf)
You can reach him at eduardo.gusmao@tum-sls.de.

## **Personal data**

Full name in native language: Eduardo Gade Gusmão
DOB: 1990-12-02
Areas of Study/Research/Leisure Interest (current research in **bold**):
	1. Biology/Medical-Related Fields:
		- **Chromatin Molecular Biology.**
		- **Computational Biology / Health Informatics / Translational Medicine.**
		- Regulatory and Epigenetic Control of Gene Regulation.
		- Multi-Omics Techniques with Single-Cell and Spatially Resolved Variants.
		- **Public Health and Populational Genomics.**
	2. Computational Fields:
		- Artificial Intelligence > Machine Learning.
		- **Deep Learning: Classical, Attention-Based & Variational Approaches.**
		- Deep Learning: Graph & Hypergraph Neural Networks.
		- **Artificial Intelligence / Deep Learning: Self-Supervision, Transfer Learning and Meta-Learning.**
		- Algorithmic Information Theory > Kolmogorov-Complexity
	3. Applied and Pure Mathematical Fields:
		- Probability Theory and Statistical Applicability.
		- Calculus, Numeric Analysis and Computational Representation.
		- **Category Theory.**
Languages:
	- Portuguese (Native).
	- English (Full Professional Proficiency).
	- Spanish (Full Professional Proficiency).
	- German (Intermediate Proficiency).
	- Italian (Intermediate Proficiency).
	- Finnish (Intermediate Proficiency).
	- French (Intermediate Proficiency).
	- Dutch (Basic Proficiency).
	- Chinese - Mandarim (Basic Proficiency).
Current number of pages **fully understood** of [James Joyce's Finnegans Wake](https://github.com/eggduzao/gusmaolab/): 77/628

## **Education**

| Period              | Degree / Title                                 | Institution & Country                                  | Honors                         |
|---------------------|------------------------------------------------|--------------------------------------------------------|--------------------------------|
| Jan 2021 – May 2023 | _Venia Docendi_ (Dosentti + Dr.-Habil.)        | **Finnish Academy of Science and Letters** *(Finland)* | Summa Cum Laude                |
| Apr 2014 – May 2018 | _Ph.D. in Molecular Biology and Public Health_ | **Harvard University** *(USA)*                         | GPA 3.96/4.0 (Summa Cum Laude) |
| Nov 2012 – Dec 2016 | _Ph.D. in Life Sciences_                       | **RWTH Aachen University** *(Germany)*                 | GPA 4.0/4.0 (Summa Cum Laude)  |
| Mar 2011 – Aug 2012 | _M.Sc. in Computer Science_                    | **Federal University of Pernambuco** *(Brazil)*        | GPA 4.0/4.0 (Summa Cum Laude)  |
| Aug 2006 – Feb 2011 | _B.Sc. in Computer Science_                    | **Federal University of Pernambuco** *(Brazil)*        | GPA 3.98/4.0 (Valedictorian)   |

## **Career History**

- _Academia:_

| Period              | Position                              | Institution & Country                                  | Department & Lab                     |
|---------------------|---------------------------------------|--------------------------------------------------------|--------------------------------------|
| Jun 2024 – Current  | _Bioinformatician_                    | **Brazilian Ministry of Health, Brazil.** *(Brazil)*   | Genome SUS                           |
| Jan 2024 – Current  | _Affiliated Professor_                | **Technical University of Munich** *(Germany)*         | TUM School of Life Sciences          |
| Jan 2023 – Dec 2023 | _Assistant Professor_                 | **Federal University of Pernambuco** *(Brazil)*        | Center of Informatics                |
| Jan 2023 – Dec 2023 | _Assistant Professor_                 | **University of Turku** *(Finland)*                    | Turku Bioscience                     |
| Apr 2019 – Dec 2022 | _Senior Postdoctoral Research Fellow_ | **Georg-August University Medical School** *(Germany)* | Pathology Institute (Papantonis Lab) |
| Feb 2018 – Mar 2019 | _Postdoctoral Research Fellow_        | **University of Cologne** *(Germany)*                  | CMMC Köln (Papantonis Lab)           |

- _Industry:_

| Period              | Position                             | Company                                     | Area/Department                          |
|---------------------|--------------------------------------|---------------------------------------------|------------------------------------------|
| Aug 2022 – Dec 2023 | _Specialist Director Advising Board_ | **BioNTech SE** *(Mainz, Germany; hybrid)*  | Institut für Infektionskrankheiten       |
| Aug 2018 – Jul 2022 | _Specialist Senior Advising Board_   | **Bayer AG** *(Wuppertal, Germany; hybrid)* | Functional Genomics & Precision Medicine |

## **Selected Skills**

**Computational & Data Analysis Skills:**
- Programming Languages: Python, UNIX Bash, AWK, R, C/C++, Java, MATLAB, SQL.
- Packages: Biopython, Pandas, NumPy, SciPy, TensorFlow, PyTorch, Samtools, Matplotlib, Seaborn, Bioconductor, DESeq2.
- Coding Strategy: Jupyter Notebooks, RMarkdown, Git, OpenBPS+SLURM+Lustre, Mamba, Nextflow, Singularity, AWS.
- File Types: AB, SR(A|R), FAST(A|Q), (S|B|CR)AM, (BIG)BED, (BIG)WIG, (V|B)CF, G(T|F)F, HIC/(M)COOL, H(DF)5.

**Selected Research and Lab Techniques:**
- Basic Techniques: (RT-)(q)PCR, Extraction & Purification, Digestion & Ligation, Gel Electrophoresis, Nanodrop.
- Intermediate Techniques: Western Blotting, IP, ChIP, EMSA, Cell Culture, Flow Cytometry, IVT, RNAi, Sanger.
- Advanced Techniques: Microarrays, CRISPR Screens, (Single-Cell) NGS, Nanopore Seq., MSMS, Cryo-EM, iPSCs, STORM.

**Selected Additional Tools and Resources:**
- General Tools: Microsoft Office, LaTeX, Trello, Slack, Zotero, Zoom, Google Meeting, LabArchives (Inventory).
- Bioinformatics Tools: BWA, Bowtie2, STAR, Cutadapt, Picard, SAMtools, GATK, FastQC, VCFtools, Cufflinks, Gprofiler.
- Databases/Browsers: GEO, NIH/NCBI/PUBMED, ENCODE, UCSC, IGV, UCSC, WashU.

**Other Relevant Skills:**
- Advanced **Grant Proposal Writing Skills**.
- Advanced Knowledge of **Modern Pedagogy**.
- Project Management Experience: Led large-scale genomic projects coordinating over 10 international institutions and managing sample collection, sample WGS, genotype and phenotype data analysis and storage teams’ efforts **exceeding $1 billion USD**.
- Currently writing two books on _"Introduction to Multi-Omics"_ and _"Aleatory Theory"_.


